[2024-08-11T09:37:24.385+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-11T09:37:24.400+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Zillow_Analytics_DAG.load_to_s3_bucket_zillowapibucket manual__2024-08-11T09:34:16.636346+00:00 [queued]>
[2024-08-11T09:37:24.405+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Zillow_Analytics_DAG.load_to_s3_bucket_zillowapibucket manual__2024-08-11T09:34:16.636346+00:00 [queued]>
[2024-08-11T09:37:24.405+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 4
[2024-08-11T09:37:24.422+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): load_to_s3_bucket_zillowapibucket> on 2024-08-11 09:34:16.636346+00:00
[2024-08-11T09:37:24.425+0000] {standard_task_runner.py:64} INFO - Started process 4090 to run task
[2024-08-11T09:37:24.427+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'Zillow_Analytics_DAG', 'load_to_s3_bucket_zillowapibucket', 'manual__2024-08-11T09:34:16.636346+00:00', '--job-id', '30', '--raw', '--subdir', 'DAGS_FOLDER/zillowrapidapi.py', '--cfg-path', '/tmp/tmp56mxvi96']
[2024-08-11T09:37:24.429+0000] {standard_task_runner.py:91} INFO - Job 30: Subtask load_to_s3_bucket_zillowapibucket
[2024-08-11T09:37:24.464+0000] {task_command.py:426} INFO - Running <TaskInstance: Zillow_Analytics_DAG.load_to_s3_bucket_zillowapibucket manual__2024-08-11T09:34:16.636346+00:00 [running]> on host ip-172-31-11-55.ap-south-1.compute.internal
[2024-08-11T09:37:24.543+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='vkkr800@gmail.com' AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='Zillow_Analytics_DAG' AIRFLOW_CTX_TASK_ID='load_to_s3_bucket_zillowapibucket' AIRFLOW_CTX_EXECUTION_DATE='2024-08-11T09:34:16.636346+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-11T09:34:16.636346+00:00'
[2024-08-11T09:37:24.544+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-11T09:37:24.553+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-11T09:37:24.554+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'aws s3 mv /home/ubuntu/response_data_11082024093720.json s3://zillow-api-bucket-1st/']
[2024-08-11T09:37:24.561+0000] {subprocess.py:86} INFO - Output:
[2024-08-11T09:37:25.107+0000] {subprocess.py:93} INFO - Completed 55.2 KiB/55.2 KiB (430.2 KiB/s) with 1 file(s) remainingmove: ../../home/ubuntu/response_data_11082024093720.json to s3://zillow-api-bucket-1st/response_data_11082024093720.json
[2024-08-11T09:37:25.170+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-11T09:37:25.171+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-11T09:37:25.190+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=Zillow_Analytics_DAG, task_id=load_to_s3_bucket_zillowapibucket, run_id=manual__2024-08-11T09:34:16.636346+00:00, execution_date=20240811T093416, start_date=20240811T093724, end_date=20240811T093725
[2024-08-11T09:37:25.244+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-11T09:37:25.253+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-11T09:37:25.253+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
